src/elements/content-sidebar/SidebarNavSign.tsx

```diff
@@ -1,66 +1,64 @@
 // @ts-ignore Module is written in Flow
 import messages from './messages';
 import './SidebarNavSign.scss';
+import SidebarNavSignButton from './SidebarNavSignButton';
+// @ts-ignore Module is written in Flow
+import { useFeatureConfig } from '../common/feature-checking';
 
-export type Props = PlainButtonProps & {
-    blockedReason?: string;
-    intl: IntlShape;
-    status?: string;
-    targetingApi?: {
-        canShow: boolean;
-        onClose: () => void;
-        onComplete: () => void;
-        onShow: () => void;
-    };
-};
-
-export const PlaceholderTooltip = ({ children }: { children: React.ReactNode }) => children;
-
-export function SidebarNavSign({ blockedReason, intl, status, targetingApi, ...rest }: Props) {
-    const isSignDisabled = !!blockedReason;
-    const isTargeted = targetingApi && targetingApi.canShow;
-    const FtuxTooltip = !isSignDisabled && isTargeted ? TargetedClickThroughGuideTooltip : PlaceholderTooltip;
-    const label = intl.formatMessage(status === 'active' ? messages.boxSignSignature : messages.boxSignRequest);
-    const buttonClassName = classnames('bcs-SidebarNavSign', { 'bdl-is-disabled': isSignDisabled });
-
-    let tooltipMessage = label;
+export type Props = PlainButtonProps;
 
-    switch (blockedReason) {
-        case 'shield-download':
-        case 'shared-link':
-        case 'shield-sign':
-            tooltipMessage = intl.formatMessage(messages.boxSignSecurityBlockedTooltip);
-            break;
-        case 'watermark':
-            tooltipMessage = intl.formatMessage(messages.boxSignWatermarkBlockedTooltip);
-            break;
-        default:
-    }
+export function SidebarNavSign({ ...rest }: Props) {
+    const {
+        blockedReason: boxSignBlockedReason,
+        onClick: onBoxClickRequestSignature,
+        onClickSignMyself: onBoxClickSignMyself,
+        status: boxSignStatus,
+        targetingApi: boxSignTargetingApi,
+        isSignRemoveInterstitialEnabled,
+    } = useFeatureConfig('boxSign');
 
     return (
-        <FtuxTooltip
-            body={intl.formatMessage(messages.boxSignFtuxBody)}
-            position={TooltipPosition.MIDDLE_LEFT}
-            shouldTarget={isTargeted}
-            title={intl.formatMessage(messages.boxSignFtuxTitle)}
-            useTargetingApi={() => targetingApi}
-        >
-            <Tooltip isDisabled={isTargeted} position={TooltipPosition.MIDDLE_LEFT} text={tooltipMessage}>
-                <PlainButton aria-label={label} className={buttonClassName} isDisabled={isSignDisabled} {...rest}>
-                    <BoxSign28 className="bcs-SidebarNavSign-icon" />
-                    <Sign16 width={20} height={20} className="bcs-SidebarNavSign-icon--grayscale" />
-                </PlainButton>
-            </Tooltip>
-        </FtuxTooltip>
+        <>
+            {isSignRemoveInterstitialEnabled ? (
+                <DropdownMenu className="dropdown-menu-sign" isResponsive constrainToWindow isRightAligned>
```

Comment:
no tooltip for this dropdown?

Reply:
the tooltip is in SidebarNavSignButton, so we keep the tooltip when hovering the sign button